yarn.lock

```diff
@@ -3633,6 +3651,19 @@ case-sensitive-paths-webpack-plugin@2.3.0:
   resolved "https://registry.yarnpkg.com/case-sensitive-paths-webpack-plugin/-/case-sensitive-paths-webpack-plugin-2.3.0.tgz#23ac613cc9a856e4f88ff8bb73bbb5e989825cf7"
   integrity sha512-/4YgnZS8y1UXXmC02xD5rRrBEu6T5ub+mQHLNRj0fzTRbgdBYhsNo2V5EqwgqrExjxsjtF/OpAKAMkKsxbD5XQ==
 
+chai-http@^4.3.0:
```

Comment:
Is this your dependency or was this introduced by the upgrade? I intended to only have this happen for developing on the repo, but I think I forgot the `--dev` flag...

Reply:
It doesn't look like we use `chai-http` in this repo.

Reply:
My fault, will be addressed in beta.1 (by stellar/js-stellar-sdk#707)